We need sysmacros.h for major/minor. Also, stropts.h has been
removed from glibc in the 2.30 version and is actually not used.

--- a/lib/gftp.h
+++ b/lib/gftp.h
@@ -48,6 +48,7 @@
 #endif
 #include <sys/wait.h>
 #include <sys/utsname.h>
+#include <sys/sysmacros.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <glib.h>
--- a/lib/pty.c
+++ b/lib/pty.c
@@ -61,10 +61,6 @@ _gftp_ptys_open (int fdm, int fds, char *pts_name)
 
 #elif HAVE_GRANTPT
 
-#if !(defined(__FreeBSD__) || defined(__NetBSD__) || defined(__APPLE__))
-#include <stropts.h>
-#endif
-
 char *
 gftp_get_pty_impl (void)
 {
